addons/mail/static/src/models/message/message.js

```diff
@@ -259,6 +259,28 @@ function factory(dependencies) {
             });
         }
 
+        cleanSubject(prefix, threadName, cleanedSubject) {
```

Comment:
I would not make a method here, especially that doesn't depend on `this` at all. Either this is a utils method (to move in proper file), or probably better you can keep this code inside `_computeIsSubjectEqualThreadName` and transform the recursive call into a loop.